jenkins/build.jpl

```diff
@@ -109,7 +111,8 @@ node("docker && builder") {
     Describe:  ${params.GIT_DESCRIBE}
     Revision:  ${params.COMMIT_ID}
     Config:    ${params.DEFCONFIG}
-    Compiler:  ${params.COMPILER}
+    C Compiler: ${params.CC}
```

Comment:
How about this instead:
```
Compiler:  ${params.CC} ${params.CC_VERSION}
```
to keep it easy to read on a single line?

Reply:
OK